src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -936,7 +936,11 @@ export class ConnectionWidget extends lifecycle.Disposable {
 						// If account was not filled in from received configuration, select the first account.
 						this._azureAccountDropdown.select(0);
 						account = this._azureAccountList[0];
-						accountName = account.key.accountId;
+						if (this._azureAccountList.length > 0) {
+							accountName = account.key.accountId;
+						} else {
+							this._logService.info('No accounts available');
+						}
```

Comment:
Info seems a bit high of a logging level for this. And do we really care about logging it anyways? Not having any accounts isn't really an error situation so as long as we're handling it gracefully that's fine IMO

```suggestion
						accountName = account?.key.accountId;
```

Reply:
Yeah I'll put it in verbose, but in general I think having more logs is better